pkg/build/path_rewriter.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package build
+
+import (
+	"net/url"
+	"path/filepath"
+)
+
+// RelativePathRewriter rewrites paths when the paths are relative paths.
+type RelativePathRewriter struct{}
```

Comment:
Are you thinking about adding any other rewriters? What is the motivation for the struct vs a function?

Reply:
I thought that I might want different rewriters. /shrug. I don't have a strong opinion.